ubuntu_from_source/Dockerfile

```diff
@@ -44,16 +36,33 @@ RUN ROOT_GIT_URL=https://github.com/root-project/root.git \
     ; fi \
     # Above lines will set ROOT_GIT_REVISION to ROOT_VERSION argument if it corresponds to a valid branch name (such as ROOT_VERSION=master),
     # otherwise it will assume its a semantic version string and try to convert it into the tag format (such as 6.26.00 -> v6-26-00)
-    && git clone --branch $ROOT_GIT_REVISION --depth=1 $ROOT_GIT_URL $ROOTSYS/source \
-    && cd $ROOTSYS/source \
-    && mkdir -p $ROOTSYS/build &&  cd $ROOTSYS/build \
-    && cmake $ROOTSYS/source \
-    -DCMAKE_INSTALL_PREFIX=$ROOTSYS \
+    && git clone --branch $ROOT_GIT_REVISION --depth=1 $ROOT_GIT_URL /tmp/source \
+    && cd /tmp/source \
+    && mkdir -p /tmp/build &&  cd /tmp/build \
+    && cmake /tmp/source \
     -DCMAKE_CXX_STANDARD=$CMAKE_CXX_STANDARD \
-    -Dbuiltin_afterimage=ON \
+    -Dgnuinstall=ON \
+    -Dsoversion=ON \
+    -DCMAKE_INSTALL_PREFIX=/usr \
+    -DCMAKE_INSTALL_BINDIR=bin \
+    -DCMAKE_INSTALL_CMAKEDIR=lib/x86_64-linux-gnu/cmake/ROOT \
+    -DCMAKE_INSTALL_DATAROOTDIR=share \
+    -DCMAKE_INSTALL_DATADIR=share/root \
+    -DCMAKE_INSTALL_DOCDIR=share/doc/root \
+    -DCMAKE_INSTALL_ELISPDIR=share/emacs/site-lisp \
+    -DCMAKE_INSTALL_FONTDIR=share/root/fonts \
+    -DCMAKE_INSTALL_ICONDIR=share/root/icons \
+    -DCMAKE_INSTALL_INCLUDEDIR=include/ROOT \
+    -DCMAKE_INSTALL_JSROOTDIR=share/root/js \
+    -DCMAKE_INSTALL_LIBDIR=lib/x86_64-linux-gnu \
+    -DCMAKE_INSTALL_MACRODIR=share/root/macros \
+    -DCMAKE_INSTALL_MANDIR=share/man \
+    -DCMAKE_INSTALL_OPENUI5DIR=share/root/ui5 \
+    -DCMAKE_INSTALL_PYTHONDIR=lib/python3/dist-packages \
+    -DCMAKE_INSTALL_SRCDIR=/dev/null \
+    -DCMAKE_INSTALL_SYSCONFDIR=/etc/root \
+    -DCMAKE_INSTALL_TUTDIR=share/root/tutorials \
     && make -j$(nproc) install \
-    && rm -rf $ROOTSYS/build $ROOTSYS/source
-
-WORKDIR /opt
+    && rm -rf /tmp/build /tmp/source
 
 CMD ["root", "-b"]

```

Comment:
You may want to use `root.exe` here to avoid errors/warnings when X11 is not properly setup inside the container.